app/code/Magento/BraintreeGraphQl/README.md

```diff
@@ -1,4 +1,9 @@
-# BraintreeGraphQl
+# Magento_BraintreeGraphQl module
 
-**BraintreeGraphQl** provides type and resolver for method additional
-information.
\ No newline at end of file
+The Magento_BraintreeGraphQl module provides type and resolver for method additional information. 
+
+## Extensibility
```

Comment:
@diazwatson This addition only instructs extension developers. As this is general instruction text which can be applicable to all the GraphQl modules, we may have to add this to all the GraphQl modules to maintain consistency if we start doing it. 

Alternatively, extension developers can go directly on the DevDocs site as that site is created specifically for that purpose. What are your thoughts?

Reply:
Hi @kalpmehta 
Yes you are right when you say this is a bit generic but at the same time I think it is better than the current version of the readme file which does not provide any instruction on how the module should be extended. 

Yes, all the modules in Magento2 will contain this paragraph "at least", this includes all the GraphQl modules.

Also, don't forget that this PR is part of a bigger task https://github.com/magento/devdocs/issues/4451 which aims not just to update the module's ReadMe but also bring consistency across them.

What do you think?

Reply:
Great, thanks @diazwatson. I think this is a good idea. Can we add this text to all the GraphQl README as well?

Reply:
Sure, it is going to be added to all modules